src/bot/scripts/card.js

```diff
@@ -27,18 +26,21 @@ let frameTops = null;
 let typeIcons = null;
 let miscIcons = null;
 
-function removeUnderscores(string) {
+const removeUnderscores = string => {
   string = string.replace(/_/g, " ").replace(/\w\S*/g, function (txt) {
```

Comment:
still havent changed this to an arrow fn

Reply:
![image](https://user-images.githubusercontent.com/54243490/71325707-5842b200-24e8-11ea-9e4d-61015720333f.png)
 ?? isnt that what an arrow fn is?